stm/stm.ml

```diff
@@ -2608,7 +2608,13 @@ let new_doc { doc_type ; iload_path; require_libs; stm_options } =
   end;
 
   (* Import initial libraries. *)
-  load_objs require_libs;
+  begin
+    try load_objs require_libs
+    with reraise when
+      (match reraise with Timeout -> true | e -> CErrors.noncritical e) ->
+         Feedback.msg_info (str "While loading initial state:");
+         raise reraise
+  end;
```

Comment:
This is probably related to @ejgallego's comment, but I feel that it is wrong to print a `msg_info` here while this info message is supposed to be attached to the error message: no guarantee about what may happen inbetween (maybe some error handling components will decide not to display the error for instance, etc.). Both the extra newline and the question of how to make it available for warnings too suggest also that this is wrong.

Reply:
Yup, I do agree.